credentials/auth.go

```diff
@@ -32,21 +34,44 @@ type netlifyConfig struct {
 	Users       map[string]netlifyUserInfo `json:"users,omitempty"`
 }
 
-var validAuthPaths = [][]string{
-	{".netlify", "config.json"},
-	{".config", "netlify"},
-	{".netlify", "config"},
-	{".config", "netlify.json"},
+func getValidAuthPaths() ([][]string, error) {
+	home, err := homedir.Dir()
+	if err != nil {
+		return nil, err
+	}
+
+	var configPaths []string
+	if goruntime.GOOS == "windows" {
+		// on windows the go xdg lib behaves a bit different than the Node.js one
+		// https://github.com/sindresorhus/env-paths/blob/de22adb240117c7aacf0365187472907a9e06872/index.js#L28
+		// https://github.com/adrg/xdg/blob/af0f1bbdcb2e9415a67e57c4125afc9daeb3ca17/paths_windows.go#L40
+		appDataDir := os.Getenv("APPDATA")
+		if appDataDir == "" {
+			appDataDir = filepath.Join(home, "AppData", "Roaming")
+		}
+		configPaths = []string{appDataDir, "netlify", "Config", "config.json"}
+	} else {
+		configPaths = []string{xdg.ConfigHome, "netlify", "config.json"}
+	}
+
+	validAuthPaths := [][]string{
+		configPaths,
```

Comment:
sorry for #lazyweb, but in https://github.com/netlify/cli/pull/1725, the change was made to save the `config.json` to different directly? and that directly is whatever you're defining over there, and that should be the default location of `config.json` moving forward (therefor it's put on top of the slice of validAuthPaths)?

Reply:
Exactly, the new location is different based on OS, usually those will be:
- Windows: `C:\Users\<user>\AppData\Roaming\netlify\Config\config.json` (`C:\Users\<user>\AppData\Roaming` can be configured via the `APPDATA` env variable).
- MacOS: `/Users/<user>/Library/Preferences/netlify/config.json`
- Linux: `/home/<user>/.config/netlify/config.json` (can be configured via `XDG_CONFIG_HOME` env variable)